scripts/jenkins/Makefile.jenkins

```diff
@@ -485,3 +485,11 @@ check-leaks:
 		echo "Credentials leaked! Please check $(PWD)/leak-check.out"; \
 		exit 1; \
 	fi
+
+h2o-docker-build:
+    docker build -t $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) -f docker/public/Dockerfile-h2o-release .
```

Comment:
nit: indentation, seems like spaces are used instead of tabs

Reply:
Replaced with tabs everywhere. Fixed. Thanks.